tests/spicy/types/unit/size-inside-size.spicy

```diff
@@ -1,27 +1,14 @@
-# @TEST-EXEC-FAIL: echo CJ2RXS7YIm4Lb+s5Y3nb0v2MVYgrmvRzZPBPQNTS4zE= | base64 -d | HILTI_DEBUG=spicy spicy-driver -d %INPUT >output 2>&1
+# @TEST-EXEC-FAIL: echo XXXX | spicy-driver -d %INPUT >output 2>&1
 # @TEST-EXEC: btest-diff output
 #
 # @TEST-DOC: Regression test for #1795: an inside &size is outside an outside &size.
```

Comment:
Is the simplified test still a regression test for #1795?

Personally, I wouldn't mind keeping the more complex test; exercising more of our machinery rarely hurts.

Reply:
> Is the simplified test still a regression test for https://github.com/zeek/spicy/issues/1795?

Yes it does and still catches the incorrect behavior, even before the fixes to `View::limit`. After the fixes we do not trigger the exact error case anymore though since we change how the `View` is modified (more correctly I'd argue).

> Personally, I wouldn't mind keeping the more complex test; exercising more of our machinery rarely hurts.

While looking into why the baseline of this test changed figuring exactly what we parsed wasn't really clear to me (both due to the need to deobfuscate the input with `base64`, and also due to the changing lengths via `&size=self.len % 3`). I do not think there is anything fundamentally interesting going on here behind this complexity. If you want I can still role back the change.

Reply:
It's ok.